packages/back-end/types/ai.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface AITokenUsageInterface {
+  id?: string;
+  organization: string;
+  numTokensUsed: number;
```

Comment:
We might want to add the daily limit to this document as well instead of a single constant used for everyone.  Especially for Enterprise customers on Cloud, I could see us wanting to increase the limit for specific orgs (or decrease the limit for abusers).

Reply:
Good call - updated